app/views/personal_details/new.html.erb

```diff
@@ -1,65 +1,21 @@
+<%= render "layouts/form_errors", resource: @personal_details %>
```

Comment:
Two nits:

1. Could we call the partial `form_errors_summary`, which is what it is?
2. Could we pass in `resource.errors`? This would make the partial a bit dumber, and its interface would be a bit nicer, eg:

```
<%= render "layouts/form_error_summary", errors: @personal_details.errors %>
```